projects/1.16/assets/evilcraft/evilcraft/lang/zh_cn.json

```
@@ -1 +1,765 @@
+  "item.evilcraft.bowl_of_promises_dusted": "装满粉末的血能碗",
+  "item.evilcraft.bowl_of_promises_tier0": "血能碗: 0级",
+  "item.evilcraft.bowl_of_promises_tier0.info": "合成中不会用尽。",
+  "item.evilcraft.bowl_of_promises_tier1": "血能碗: 1级",
+  "item.evilcraft.bowl_of_promises_tier1.info": "合成中不会用尽。",
+  "item.evilcraft.bowl_of_promises_tier2": "血能碗: 2级",
+  "item.evilcraft.bowl_of_promises_tier2.info": "合成中不会用尽。",
+  "item.evilcraft.bowl_of_promises_tier3": "血能碗: 3级",
+  "item.evilcraft.bowl_of_promises_tier3.info": "合成中不会用尽。",
+  "item.evilcraft.dull_dust": "暗尘",
+  "item.evilcraft.dull_dust.info": "看起来相当暗沉，对吧?",
+  "item.evilcraft.blood_waxed_coal": "血蜡煤",
+  "item.evilcraft.blood_waxed_coal.info": "双倍燃烧效率",
+  "item.evilcraft.ender_tear": "末影人之泪",
+  "item.evilcraft.ender_tear.info": "末影人的稀有掉落。",
+  "item.evilcraft.blood_potash": "染血骨粉",
+  "item.evilcraft.blood_potash.info": "双倍效率",
+  "item.evilcraft.bucket_eternal_water": "永恒水桶",
+  "item.evilcraft.bucket_eternal_water.info": "无限水之源",
+  "item.evilcraft.promise_acceptor_iron": "铁之诺言接收器",
+  "item.evilcraft.promise_acceptor_gold": "金之诺言接收器",
+  "item.evilcraft.promise_acceptor_diamond": "璀璨诺言接收器",
+  "item.evilcraft.sceptre_of_thunder": "雷电权杖",
+  "item.evilcraft.sceptre_of_thunder.info": "使用会使天气变为雷雨天，一次性道具。",
+  "item.evilcraft.blood_orb": "血晶球",
+  "item.evilcraft.blood_orb_empty": "空晶球",
+  "item.evilcraft.blood_orb_filled": "血晶球",
+  "item.evilcraft.origins_of_darkness": "黑暗起源",
+  "item.evilcraft.origins_of_darkness.info": "通过喂动物一个黑苹果然后快速向其死后的灵魂之门内扔一本书得到",
+  "item.evilcraft.darkened_apple": "黑苹果",
+  "item.evilcraft.darkened_apple.info": "我才不会吃它，把它喂给动物试试。",
+  "item.evilcraft.veined_scribing_tools": "静脉划线工具",
+  "item.evilcraft.blood_wand_cap": "血杖端",
+  "_item.Wand.blood.cap": "Blood Banded",
+  "item.evilcraft.effortless_ring": "迅捷指环",
+  "item.evilcraft.effortless_ring.info": "行走速度更快更高效。",
+  "item.evilcraft.condensed_blood": "凝结血液",
+  "item.evilcraft.condensed_blood.info": "放入机器中当作血液使用.",
+  "item.evilcraft.primed_pendant": "装载吊坠",
+  "item.evilcraft.primed_pendant.info": "产生选定的药水效果。",
+  "item.evilcraft.primed_pendant.potion": "§7效果: %s (%s)",
+  "item.evilcraft.golden_string": "金线",
+  "item.evilcraft.biome_extract": "群系要素瓶",
+  "item.evilcraft.biome_extract.empty": "群系要素空瓶",
+  "item.evilcraft.biome_extract.info": "将目标群系转换为所含生物群系,在相应生物群系中的环境存储器上制作.",
+  "item.evilcraft.biome_extract.info.content": "包含: %s",
+  "item.evilcraft.environmental_accumulation_core": "环境存储器核心",
+  "item.evilcraft.environmental_accumulation_core.info": "破坏常规的环境存储器时可以获得。",
+  "item.evilcraft.mace_of_destruction": "毁灭之锤",
+  "item.evilcraft.garmonbozia": "痛苦之源",
+  "_item.evilcraft.garmonbozia.info": "\"Pain and Sorrow\"",
+  "item.evilcraft.poison_bottle": "毒液瓶",
+  "item.evilcraft.vengeance_essence": "复仇精华",
+  "item.evilcraft.vengeance_essence.info": "杀死复仇之魂掉落",
+  "item.evilcraft.vengeance_essence_materialized": "物化复仇之魂",
+  "item.evilcraft.piercing_vengeance_focus": "复仇刺穿棱镜",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "item.evilcraft.piercing_vengeance_focus": "穿刺复仇之焦",
```
老版本翻译简直就是错漏百出。